static/js/ducks/source.js

```diff
@@ -14,6 +14,8 @@ const ADD_CLASSIFICATION = "skyportal/ADD_CLASSIFICATION";
 
 const DELETE_CLASSIFICATION = "skyportal/DELETE_CLASSIFICATION";
 
+const ADD_TNS = "skyportal/ADD_TNS";
```

Comment:
Maybe better to use `ADD_SOURCE_TNS` and `addSourceTNS`?

Reply:
good idea.